app/models/application_merits_task/involved_child.rb

```diff
@@ -1,5 +1,6 @@
 module ApplicationMeritsTask
   class InvolvedChild < ApplicationRecord
     belongs_to :legal_aid_application
+    has_many :application_proceeding_type_involved_children, dependent: :destroy
```

Comment:
I think this model should be moved out of the `ApplicationMeritsTask` namespace as it's also going to be using by `ProceedingMeritsTask`. What do you think @colinbruce ?